src/frontend/src/storage/index.ts

```diff
@@ -91,6 +92,8 @@ const updateStorage = (
   doWrite ||= pruned;
 
   if (doWrite) {
+    await writeIndexedDB(prunedAnchors);
```

Comment:
The localStorage calls are synchronous anyway so nothing to be resolved there

Reply:
If `writeIndexedDB` throws an exception, `writeLocalStorage` won't be called (if I get it right). So, ss that an issue?

For example, both must succeed to be consider the process as ok:

```
await Promise.all([writeIndexedDB(), async () => writeLocalStorage()]);
```

Both must be executed regardless if one fails:

```
await Promise.allSettled([writeIndexedDB(), async () => writeLocalStorage()]);
```

It does not matter if idb fails:

```
try {
   await writeIndexedDB();
} finally {
   writeLocalStorage();
}
```

etc.

Not sure it's more clear though, maybe it is no concern or I don't get it though.

Reply:
Mh that's a good question. I think it's fine _not_ to write to localStorage if indexeddb fails; better even maybe. Otherwise the storages will get out of sync. And if local storage fails I'm not sure what we'd do; we wouldn't undo the idb write anyway.

Reply:
> will get out of sync.

Makes sense